kernel/packages/decentraland-ecs/src/decentraland/math/Vector3.ts

```diff
@@ -97,6 +97,7 @@ export class Vector3 {
    * @param normal - direction of the normal
    * @returns the angle between vector0 and vector1
    */
+  // Unity returns a different result -> unsigned, smallest angle: https://docs.unity3d.com/ScriptReference/Vector3.Angle.html
```

Comment:
This comment here doesn't break the method documentation? (when vscode intellisense displays the documentation of the method you are writing)

Reply:
Done